packages/core/src/external/canvas/note.ts

```diff
@@ -0,0 +1,85 @@
+import CanvasSDK from "./index";
+import { generateFakeBundleFemale, generateFakeBundleMale } from "./data";
+import { out } from "../../util/log";
+const { log } = out("[CANVAS NOTE]");
+
+export async function createFullNote({
+  canvas,
+  canvasPatientId,
+  patientA,
+  patientB,
+}: {
+  canvas: CanvasSDK;
+  canvasPatientId: string;
+  patientA: boolean;
+  patientB: boolean;
+}) {
+  try {
+    const canvasPractitioner = await canvas.getPractitioner("Wilson");
+    const canvasPractitionerId = canvasPractitioner.id;
+
+    const canvasLocation = await canvas.getLocation();
+    const canvasLocationId = canvasLocation.id;
+
+    const canvasEncounter = await canvas.getFirstEncounter(canvasPatientId);
+    const canvasEncounterId = canvasEncounter.id;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const canvasNoteId = (canvasEncounter.extension as any)[0].valueId;
```

Comment:
Let's please add code to avoid runtime errors when we access fixed positions on arrays.

In this case, we could either store the result of `...[0]` somewhere and check it before accessing `valueId`, or add question mark `...[0]?.valueId` and check the result of that for emptyness.